Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -66,6 +67,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if (isset($params['master'])) {
             $params = $params['master'];
         }
+        if (isset($params['shards'])) {
+            $shards = $params['shards'];
+            // Default select global
+            $params = $params['global'];
+            if ($input->getOption('shard')) {
+                foreach ($shards as $shard) {
+                    if ($shard['id'] == $input->getOption('shard')) {
```

Comment:
Nop cause `$input->getOption('shard')` is interpreted as a string, and `$shard['id']` is an integer